core/base/inc/TROOT.h

```diff
@@ -26,8 +26,8 @@
 //////////////////////////////////////////////////////////////////////////
 
 #include "TDirectory.h"
-#include "TList.h"
-#include "RConfigure.h"
+// #include "TList.h"  // included in TDirectory.h
```

Comment:
But it is just directly here, so we ought to keep it.

Reply:
See TDirectory.h - "TList.h" included only without `R__LESS_INCLUDES`. Means in the dev build `TList` is just forward declarations. I just want to avoid same construct here.